cpp/src/Ice/UserException.cpp

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#include "Ice/UserException.h"
+#include "Ice/InputStream.h"
+#include "Ice/OutputStream.h"
+
+using namespace std;
+using namespace Ice;
+using namespace IceInternal;
+
+std::string_view
+Ice::UserException::ice_staticId() noexcept
+{
+    static constexpr string_view typeId = "::Ice::UserException";
+    return typeId;
+}
+
+void
+Ice::UserException::_write(Ice::OutputStream* os) const
```

Comment:
Don't need for `Ice::` as we have `using namespace Ice`

Reply:
Removed